.changesets/fix_bnjjj_fix_4348.md

```diff
@@ -0,0 +1,5 @@
+### Fix fragment spread response formatting ([Issue #4348](https://github.com/apollographql/router/issues/4348))
+
+When you had an inline fragment on an union type in a fragment spread, the response formatting was wrong.
```

Comment:
Hmm, "Response formatting was wrong" doesn't really explain the impact to me.   What was the impact on the user and that they would have seen?  How does this manifest?